### PR TITLE
Update CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by mrmakadia94**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@mrmakadia94 the code of conduct is taken directly from https://github.com/ContributorCovenant/contributor_covenant, so you're best submitting a PR there and we can update when they release and update.  The changes for the Contributing.md file look good though.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at engage@xwp.co. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing or otherwise unacceptable behavior may be reported by contacting the project team at engage@xwp.co. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (jeffpaul) [inline CODE_OF_CONDUCT.md]: If you roll back this change, then the single change for Contributing.md looks good for merge.

**Review by jeffpaul — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mrmakadia94) [reply CODE_OF_CONDUCT.md]: okay
i roll back this change

Comment (mrmakadia94) [reply CODE_OF_CONDUCT.md]: Hello @jeffpaul 
i rollback changes for this file.
Now only CONTRIBUTING.md file for merge.

Thanks.

**Review by jeffpaul — APPROVED** [review]
looks good to me

**Pull request merged**